bessctl/conf/testing/run_module_tests.bess

```diff
@@ -84,76 +86,99 @@ path = os.path.dirname(os.path.realpath(sys.argv[0]))
 for filename in glob.glob(path + "/conf/testing/module_tests/*.py"):
```

Comment:
Sure. I'm mostly just trying to avoid using globals when we don't need them. I was thinking of something that just eats the code starting at "## RUN TEST LOOP##" e.g., `do_all_tests()`. It would have its own local `did_tests_fail`-type flag that it would return.